web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -584,7 +587,8 @@
           doCycleIn200();
         });
       },
-      logError: settings.logError,
+      logError: function() { settings.logError.apply(this, arguments) },
```

Comment:
I'm curious why this was necessary. Do the arguments not get passed along properly without it? Or was there some other reason?